core/settings/manager.php

```diff
@@ -150,6 +150,10 @@ public static function get_settings_managers_config() {
 		foreach ( self::$settings_managers as $name => $manager ) {
 			$settings_model = $manager->get_model_for_config();
 
+			if ( ! $settings_model ) {
```

Comment:
Is there a real case where there isn't a settings model?

Reply:
There is no real case, and if you see commits, you will see that it's part of test commit, anyway we have issue with the tests.